server/bin/docker/genCerts.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+WORKING="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+. ${WORKING}/deploy-env.sh
+
+PASSPREFIX=pass:
+KEYPASS=password
+PASSWORD=$PASSPREFIX$KEYPASS
+
+if [ ! -d ${SRC_DIR}/pki ]; then
+	mkdir ${SRC_DIR}/pki
+	pushd ${SRC_DIR}/pki
+
+	#Create a private key
+
+	openssl genrsa -des3 -out CA.key -passout ${PASSWORD} 4096
```

Comment:
It appears that this is copied from our [documentation] (https://nationalsecurityagency.github.io/timely/docs/#ssl-setup).  We should update that too.